README.md

```diff
@@ -1,32 +1,242 @@
-appropriate to help with the reader's orientation with the material and future
-referencing.
 
-## Topic 2
+## Junit Dependency
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+We’ll add a unit test to our project, and then we’ll learn how to add the Junit
+dependency to make it work. 
+
+The `Main` class has been updated with a new method `greet()`:
+
+```java
+package org.example;
+
+public class Main {
+
+    public static String greet(String name) {
+        return "Hello, " + name + "!";
+    }
+
+    public static void main(String[] args) {
+        System.out.println("Hello!");
+    }
+}
+```
+
+Right-click in the `Main` class and select **Generate | Test** to generate
+a Junit test class.  Check the option to generate a test method for the `greet()` method:
+
+![MainTest Junit class](https://curriculum-content.s3.amazonaws.com/6002/maven-dependencies/newtest.png)
+
+The `MainTest` class should test the static `Main.greet()` method as shown:
+
+```java
+package org.example;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MainTest {
+
+    @org.junit.jupiter.api.Test
+    void greet() {
+        String expected = "Hello, John!";
+        String actual = Main.greet("John");
+        assertEquals(expected, actual);
+    }
+}
+```
+
+Unfortunately, `MainTest` won't run because the
+Junit library can't be found.  We need to add a dependency to `pom.xml`
+to tell Maven to download the Junit library.
+
+![Junit missing error](https://curriculum-content.s3.amazonaws.com/6002/maven-dependencies/errors.png)
+
+Hover over the word "junit" in the import statement or the test annotation
+to see the error.   Click "more actions" to add the most updated version of Junit
+to the class path:
+
+![Add Junit to classpath](https://curriculum-content.s3.amazonaws.com/6002/maven-dependencies/junitclasspath.png) 
+
+The errors disappear in `MainTest` and the test class can be executed.
+
+1. Run the `MainTest` class and confirm the unit test passes.
+2. Open the Build Tool Window by clicking the  **Build** icon on the lower toolbar. 
+   There are quite a few tasks required for the **test** phase of the lifecycle!
+
+### pom.xml
+
+The `pom.xml` file was automatically updated when Junit was added to the classpath.
+The  file has a new dependency element containing the coordinates of the Junit
+library:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>example-app</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
Suggestion: Maybe add an XML comment here (or even above the dependency tag) to make it a little clearer to the students on where at exactly this was added in the pom.xml?

Reply:
Updated xml with comment